app/facades/rev_it_up_facade.rb

```diff
@@ -24,4 +24,12 @@ def price
   def category
     @data[:categories][0][:title]
   end
+
+  def lat
```

Comment:
Lat and long to be used to imbedded map image and possible directions to the restaurant chosen from current location.